common/src/Admin.php

```diff
@@ -111,6 +111,17 @@ public function options_page() {
     public function invite_error_notices() {
         global $wpdb;
 
+        $table_exists = $wpdb->get_var("
+            SELECT COUNT(*)
```

Comment:
1. If you care only about whether or not a row is found, use `SELECT 1 ... LIMIT 1`. This way the database can stop at the first matching row instead of having to count all of them. This is not going to be very relevant in this case, but it's a good habit to write your does-it-exist queries like this.
2. Please indent the `WHERE` clause as below (`AND` is a part of the `WHERE`, not a separate clause).
3. _Always_ use an escaping mechanism for inserting variables into queries.

Eg:

~~~php
$wpdb->get_var($wdbp->prepare(
  "
    SELECT 1
    FROM information_schema.tables
    WHERE
      table_schema = %s
      AND table_name = %s
    LIMIT 1
  ",
  [
    $wpdb->dbname,
    $this->plugin->getInviteErrorsTable(),
  ],
));
~~~